README.rst

```diff
@@ -11,6 +11,27 @@ managed Infrastructure as a Service (IaaS), and Networks Functions
 Virtualization (NFV) use cases.
 
 
+Package source
+==============
+Formula support OpenContrail as well as Juniper Contrail package repository in the backend.
+
+Differences withing the configuration and state run are controlled by
+``opencontrail.common.vendor: [opencontrail|juniper]`` pillar attribute.
+
+Default value is set to ``opencontrail``.
+
+Juniper releases tested with this formula:
+ - 3.0.2.0-51
```

Comment:
I would not reference specific versions of Juniper, because then we have to update readme even if content is same. It will work with all 3.0.2.x versions.